src/asprof.cpp

```diff
@@ -53,3 +55,17 @@ DLLEXPORT asprof_error_t asprof_execute(const char* command, asprof_writer_t out
 DLLEXPORT asprof_thread_local_data* asprof_get_thread_local_data(void) {
     return ThreadLocalData::getThreadLocalData();
 }
+
+DLLEXPORT asprof_user_jfr_key asprof_create_user_jfr_key(const char *name) {
+    return UserEvents::registerEvent(name);
+}
+
+DLLEXPORT int asprof_emit_user_jfr(asprof_user_jfr_key key, const uint8_t *data, size_t len) {
```

Comment:
Will it ever be needed to record events with stack traces?
I initially thought event will have a bitmask of standard fields that should be recorded along with user-provided data. These fields may include `threadId` and `stackTrace`.

(It's more like a question for discussion, not necessarily for this PR).